### ISSUE TITLE
"Keep me logged in" checkbox

### ISSUE THREAD

**Issue by Naviary2** [Infinite-Chess/infinitechess.org#340]
With the latest addition of session renewal for as long as you are active, this is as perfect at time as ever to add a checkbox on the login screen "Keep me Logged In".

If this box is checked when the user clicks login, it will include that boolean when it sends the login credentials.

On the server-side, we reduce the expiry time of session tokens to a few days (located in config.js), and we add another variable for storing the duration of "extended sessions", which will last six months (if never renewed). If this "keep me logged in" boolean is present, loginController.handleLogin() will pass that as an argument to createNewSession(), which should trigger it to use the longer expiry time when issuing the new session.

Generated session tokens inside refreshTokenObject.addTokenToRefreshTokens() also need to store this boolean of whether or the session is of the extended type. When the session is renewed within sessionManager.renewSession(), we renew it for the same six months if that boolean is present on the session token.

**Comment by Adarsh1826** [comment]
Can i start working on this issue

**Comment by Naviary2** [comment]
Yeah, go ahead! Desc is a bit outdated, some of the refresh token logic has since changed.

**Comment by Naviary2** [comment]
Added backend support for this in 6976af69c57b522fb7a050cb31622e419c4c2444.
Login page is yet to be designed on the `redesign` branch.
